setup.cfg

```diff
@@ -191,3 +191,6 @@ exclude_lines =
     raise AssertionError
     raise NotImplementedError
     if __name__ == '__main__':
+omit =
```

Comment:
Would it be suitable to have these omitted already in the `[coverage:run]` section, as opposed to here in the `report` section?  I'm no expert, so ...

Reply:
The documentation suggests that non-Python files are already skipped, but I'll add it there as well just to be safe.